component/src/main/ballerina/mongodb/actions.bal

```diff
@@ -14,72 +14,57 @@
 // specific language governing permissions and limitations
 // under the License.
 
-documentation {
-    The Caller Actions for MongoDB databases.
-}
+# The Caller Actions for MongoDB databases.
 public type CallerActions object {
-    documentation {
-        The find operation implementation which selects a document in a given collection.
 
-        P{{collectionName}} The name of the collection to be queried
-        P{{queryString}} Query to use to select data
-        R{{}} `json` result from the find operation or `error` if an error occurs
-    }
+    # The find operation implementation which selects a document in a given collection.
+
```

Comment:
Need to add a `#` here.

Reply:
Fixed with eaf29c008b28b99f792aebbc286877b1bc3fc6b3